doc/md_src_files/040_stakeholders-scope.md

```diff
@@ -23,12 +23,61 @@ Relatedly, C-level executives and public policy professionals often make, shape,
+The unit of work may be different for different stakeholders.
+The units of work can also change as the vulnerability response progresses through a stakeholder's process.
+We elucidate these ideas with the following examples.
 
-### Supplying Patches
+### Supplier Units of Work
+
+On the input side of the Supplier process, Suppliers typically receive reports of vulnerabilities in one or more versions of their product. 
+Part of the Supplier's task on initial report intake is to resolve the initial report into a set of products and versions that are affected by the reported vulnerability. 
+On the output side, Suppliers provide fix bundles for affected products.
+
+Our working assumption is that for SSVC purposes, the Supplier's unit of work is the tuple of (vulnerability, affected product). 
+This implies the need for Suppliers to be able to resolve whatever they receive to that level of granularity in order to make best use of SSVC.
+
+There are a variety of ways a Supplier might need to resolve the set of affected products. For example they might
+
+* recognize, on further investigation of the initial report, that additional versions of the product are affected
+* discover that other products are affected due to code sharing or even consistent programmer error across products
+* receive reports of vulnerabilities in third party libraries they include in one or more of their products
+* receive fix bundles for third party libraries used in one or more of their products (where a fix bundle might resolve multiple vulnerabilities or add new features)
+
+We note that SBOM seems well-placed to aid in that resolution process for the third party library scenarios.
+
+
+### Deployer Units of Work ###
+
+Deployers are usually in the position of receiving fix bundles from their Suppliers for products they have deployed. 
+They are then faced with the decision to deploy the fix bundle to a particular instance or not. 
+Whether or not they have the option of deploying only part of a fix bundle depends on whether the Supplier has engineered their release process to permit that degree of flexibility. 
+For example, if service packs are fix bundles, the Supplier might choose to release individually deployable fixes as well.
 
+The vulnerability management process for deployers has at its core the collation of data including
+* an inventory of deployed instances of product versions
+* a mapping of vulnerabilities to fix bundles
+* a mapping of fix bundles to product versions
+The first of these must be collected by the deployer, while the latter two most often originate from the product Supplier.
```

Comment:
Add:
> Managing this information is generally called **asset management**. 
The relationship between SSVC and asset management is discussed further in [Relationship to asset management](#relationship-to-asset-management).

Reply:
fixed text in 1955582b53bea3c5550dee83ba4629f8f8e95f10 but I don't know how to do proper cross-references in the markdown format if that's a thing.